qt/icons/src/CharIconPainter.cpp

```diff
@@ -43,7 +43,9 @@ void CharIconPainter::paintIcon(IconicFont *iconic, QPainter *painter,
   // Set some defaults so it doesn't fail later if nothing was set
   QString color("black");
```

Comment:
I think this is fine but can be define our defaults more prominently rather than inline in the code? Better has named anonymous-namespace variables.

Reply:
That is a sound suggestion.